### PR TITLE
Implemented the LoadProgramBanList method. Closed #49

### LINKED ISSUE #49: Implement the LoadProgramBanList method
Implement the LoadProgramBanList method in the ProgramBanListService class. Add different loading options: from file and from server.

### PR DESCRIPTION
**Pull request by London3149-train**

### CODE REVIEW

File: user-monitoring-gui/Services/ConsoleUserInterface.cs
Comment (TopAcademy2022) [inline user-monitoring-gui/Services/ConsoleUserInterface.cs:1]: Revert this file.

File: user-monitoring-gui/Services/Interfaces/IServerRequest.cs
Comment (TopAcademy2022) [inline user-monitoring-gui/Services/Interfaces/IServerRequest.cs:1]: Revert this file to main (HEAD).

File: user-monitoring-gui/Services/RunProgramReader.cs
Comment (TopAcademy2022) [inline user-monitoring-gui/Services/RunProgramReader.cs:1]: Revert this file.

**Review by TopAcademy2022 — CHANGES_REQUESTED** [review]
Why main (HEAD) files was edited?

**Review by HecuseS — DISMISSED** [review]

**Review by cany8042 — DISMISSED** [review]

**Review by Andrey31ts — DISMISSED** [review]

**Review by cany8042 — DISMISSED** [review]

File: user-monitoring-gui/Services/ProgramBanListService.cs [hunk 1]
```diff
@@ -1,17 +1,60 @@
...
 using user_monitoring_gui.Services.Interfaces;
 
 namespace user_monitoring_gui.Services
 {
     public class ProgramBanListService : IProgramBanListService
     {
-        public bool Save(ProgramBanList programBanList)
+
+        private IServerRequest _serverReqwest;
+
```
Comment (TopAcademy2022) [inline user-monitoring-gui/Services/ProgramBanListService.cs:10]: Remove space.

File: user-monitoring-gui/Services/ProgramBanListService.cs [hunk 2]
```diff
@@ -1,17 +1,60 @@
...
         }
 
-        public bool Load()
+        /*!
+         * @brief Load method reads data for ProgramBanList
+         * @param programBanList - the class is passed
+         * @param dataStorageArea - the class is passed
+         * @return True if data was received, false otherwise.
+         */
+
```
Comment (TopAcademy2022) [inline user-monitoring-gui/Services/ProgramBanListService.cs]: Remove space.

File: user-monitoring-gui/Services/ProgramBanListService.cs [hunk 3]
```diff
@@ -1,17 +1,60 @@
...
+                foreach( var line in lines )
+                {
+                    programBanList.AddProgram( line );
+                }
+                return true;
+
+            case DataStorageArea.SERVER:
+
+                var byat = this._serverReqwest.LoadData( programBanList.GetType()).GetData();
```
Comment (TopAcademy2022) [inline user-monitoring-gui/Services/ProgramBanListService.cs]: Rename variable.

**Review by TopAcademy2022 — CHANGES_REQUESTED** [review]

**Review by HecuseS — DISMISSED** [review]

**Review by cany8042 — APPROVED** [review]

**Review by HecuseS — APPROVED** [review]

**Review by xs887z — APPROVED** [review]

**Review by SugarSalt9 — APPROVED** [review]

**Review by Andrey31ts — APPROVED** [review]

**Review by Mi-stik31 — APPROVED** [review]

**Review by TopAcademy2022 — APPROVED** [review]

**Pull request merged**